### DISCUSSION TITLE
No module named `ase.filters` when using `StructOptimizer`

### DISCUSSION THREAD

**Discussion by ColdSnaap** [CederGroupHub/chgnet#154] [Q&A] [answered]
It seems like there is no ase.filters module as I checked the ase source code but they have it in their documentation. Kind of weird.

**Comment by bowen-bd** [comment, marked as answer]
Please refer to https://github.com/CederGroupHub/chgnet/issues/107 and https://github.com/CederGroupHub/chgnet/issues/127

**Reply by ColdSnaap** [reply]
Thanks!
